composer.json

```diff
@@ -32,29 +32,28 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.2",
-        "cebe/markdown": "^1.2@dev",
+        "ext-filter": "*",
         "httpsoft/http-message": "^1.1.6",
         "psr/container": "^2.0",
         "psr/http-factory": "^1.0",
         "psr/http-message": "^2.0",
         "psr/http-server-handler": "^1.0",
         "psr/http-server-middleware": "^1.0",
         "psr/log": "^3.0.2",
+        "psr/simple-cache": "^3.0",
         "symfony/console": "^7.3.1",
         "vlucas/phpdotenv": "^5.6",
+        "yiisoft/assets": "^5.1",
+        "yiisoft/cache": "^3.1",
         "yiisoft/cache-file": "^3.2",
         "yiisoft/config": "^1.6",
         "yiisoft/data": "^1.0.1",
         "yiisoft/data-response": "^2.1",
         "yiisoft/definitions": "^3.4",
-        "yiisoft/di": "^1.4",
```

Comment:
Keep `yiisoft/di`. It is used by fact.

Reply:
Why? Yes it's used but not in app directly.

Reply:
We use it directly in container definitions.

Reply:
For example, reset state is Yii DI specific feature. By fact, we use definitions of Yii DI.

Reply:
@vjik where do you see reset usage right now?

Reply:
FYI: also app template doesn't have `yiisoft/di` dependency.

Reply:
> FYI: also app template doesn't have `yiisoft/di` dependency.

Need to add it too.

Reply:
@vjik but it's `Psr\Container\ContainerInterface`, not yiisoft/di.

Reply:
> @vjik but it's `Psr\Container\ContainerInterface`, not yiisoft/di.

Resetter is Yii DI specific feature

Reply:
> Resetter is Yii DI specific feature

Yes, but it depends on runner. By default it's unused feature.

Reply:
We use in application configuration Yii DI definitions.